### PR TITLE
Container Identifiers

### PR DESCRIPTION
**Pull request by dimitribouniol**
**These changes are now available in [4.2.0](https://github.com/vapor/apns/releases/tag/4.2.0)**

To help users who need to configure a production and development container, added two new containerIDs: `production` and `development`. Also added documentation and clarified how the `default` ID is used, which isn't always the default depending on how `use` is called.

Note: Since this is new, I went with `development` instead of `sandbox` here because all of Apple's documentation refers to it as the development environment. We should probably add `.development` and deprecate `.sandbox` in APNSwift as well.

### CODE REVIEW

File: Sources/VaporAPNS/APNSContainers.swift [hunk 1]
```diff
@@ -50,7 +50,90 @@ public final class APNSContainers: Sendable {
 }
 
 extension APNSContainers {
-
+    /// Configure APNs for a given container ID.
+    ///
+    /// You must configure at lease one client in order to send notifications to devices. If you plan on supporting both development builds (ie. run from Xcode) and released builds (ie. TestFlight/App Store), you must configure at least two configurations:
+    ///
+    /// ```swift
+    /// let apnsKey = ... // The .p8 file as a string.
```
Comment (0xTim) [inline Sources/VaporAPNS/APNSContainers.swift:58]: Let's add a note here to suggest passing it in as an environment variable and not hardcoding it in the source code

**Review by 0xTim — CHANGES_REQUESTED** [review]
One minor addition

File: Sources/VaporAPNS/APNSContainers.swift (hunk 1 above)
Comment (dimitribouniol) [reply Sources/VaporAPNS/APNSContainers.swift:58]: Good callout, I’ll add a section for security considerations.

Comment (dimitribouniol) [reply Sources/VaporAPNS/APNSContainers.swift]: Updated!

**Review by 0xTim — APPROVED** [review]
Thanks!

**Pull request merged**